### ISSUE TITLE
.scss autocompletion gone in sublime text 3

### ISSUE THREAD

**Issue by mpitz** [P233/Syntax-highlighting-for-Sass#78]
After the last update the autocompletion in the .scss files dosn't work anymore but in the .sass files.

System: Windows 10, Sublime Text 3

**Comment by lisunovdv** [comment]
Just re-install the plugin. It works for me (totally remove package and install it from scratch)

**Comment by mpitz** [comment]
Didn't work. After removing Emmet too the autocompletion works as expacted, but that's not an option.

**Comment by Lopol2010** [comment]
For me both scss and sass not work today.

System: Windows 10, Sublime Text 3

**Comment by Lopol2010** [comment]
Ok sorry, just read the _New Features_ section
**Removed built-in completions**

**Comment by Lopol2010** [comment]
**Emmet Css Snippets** appears to be have absolutely the same autocompletions and:
Works with default CSS package, LESS, and **Syntax Highlighting for Sass**

**Comment by mpitz** [comment]
Right, I also missed the "Feature" description. Hm I will give the Emmet CSS Snippets a chance.

**Comment by Lopol2010** [comment]
This pack work perfect for me: 
Emmet Css Snippets - for props like color, height etc.
 SASS Snippets - for functions such as rgb()...
Sass - you know

**Comment by mpitz** [comment]
It's really annoying that the built in completion is gone!

For example in SCSS:
typing "colo" doesn't complete "color"
typing "posi" doesn't complete "position" further "abs" doesn't complete "absolute"

And very annoying:
all defined sass/scss variables aren't in the completion

If you switch in a scss document to sass than all work's like before but the curly brackets are highlighted/marked.

Please readd the completion!

**Comment by minstyle** [comment]
I have the same problem with SCSS as mpitz described. Please fix it.

**Comment by ThaoD5** [comment]
The _feature_ 

> Removed built-in completions

 is really a bad "feature" (even if it's **not**)... 

Is there a chance to roll back on this one ? I don't really see the point of removing this **important** feature (because having it is a feature, removing isn't) 👎

**Comment by Anthodpnt** [comment]
Hi @P233,

Thanks for the good work on the **previous** releases...
I think we're a lot to have some questions about this 🎉 **brand new feature** 🎉  that removes the built-in completion of Sublime Text. Here are mine and I hope you have relevant answers to them:

1. Why do you need to remove the built-in completion for your package ?
2. Why do we need to install 2 more packages because of this feature ?
3. Don't you think a package should improve Sublime Text **without** removing one of it's most used built-in feature ?
4. When did you think it was even a good idea ?

I can't imagine the number of hours developers around the world should spent right now trying to figure out why their completion isn't working anymore because of your last release instead of writing code for maybe a next amazing website, application,...

No one here understand the pros of removing a core feature of Sublime Text and the advantage for your package I guess. Personally I really hope it's an April Fool's joke and that you made a mistake reading the date this morning when getting up.

Best regards,
Anthony

**Comment by qgates** [comment]
As a temporary workaround, pull the `css_completions.py` out of your `Packages\CSS.sublime-package' and modify line 449 (build 3143, may vary on newer builds) to:

```
css_selector_scope = "source.css - meta.selector.css, source.scss - meta.selector.scss"
```
Drop the file in your Sublime Packages\User folder, restart sublime and good old CSS completions should be back.

From [this thread](https://github.com/P233/Syntax-highlighting-for-Sass/issues/72#issuecomment-360785336), it seems that author @P233 is incorporating standard CSS completions back, but the above workaround seems to work for now :-)

**Comment by mikevaux** [comment]
Guys, I think the development community would benefit from slightly less grumbling and sarcasm when reporting issues. Let's remember that we are all benefitting from @P233 's work for _free_.. If issues are important to you, then definitely report them, but maybe with a bit of grace for things you  don't understand or possible errors :-)

**Comment by mikevaux** [comment]
p.s. @qgates that looks helpful!

**Comment by P233** [comment]
Hi,

In terms of the "Removed built-in completions", I was comparing with the previous version of this package which happened before this package takes over the package name of "Sass".

In the ST2 version, there was a [feature](https://github.com/P233/Syntax-highlighting-for-Sass/tree/ST2#built-in-completion-packages) to filter completions based on scopes. But it increased the difficulty and time spent to maintain this package. When ST3 introduced the [new syntax definition format](https://www.sublimetext.com/docs/3/syntax.html), I decided to give it a shot.

Since I don't use ST as my primary editor (except reading code), I want to abandon the completion feature and rely it on the Emmet plugin; then I could focus on the highlighting accuracy and save my free time.

<img width="405" alt="screen shot 2018-02-05 at 9 12 08 pm" src="https://user-images.githubusercontent.com/1818679/35808109-c1db9d98-0abf-11e8-8edb-d02994ffa363.png">

And you could see how I improved accuracy in the new version:

<img width="635" alt="screen shot 2018-02-05 at 9 22 49 pm" src="https://user-images.githubusercontent.com/1818679/35808126-c8f72fe8-0abf-11e8-87c2-f03dbf708787.png">

Now I am planning to reorganise scopes which could fix two existing bugs and bring back the completion feature with accuracy popup position. Please bear with me for a while longer.

**Comment by dortamiguel** [comment]
Emmet completions are not working

**Comment by blakewatson** [comment]
@P233 Would the adjustments you are talking about bring back auto-completing text that appears in the file (like class names)? I have no problem with using Emmet for props and values, but I use auto-completion as an accessibility feature (I have difficulty typing) to help me type repeated/similar selectors and such.

**Comment by yordanzhelevdev** [comment]
Bump

**Comment by krispind** [comment]
Bump
